docs/services/motion/constraints.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Configure Motion Constraints"
+linkTitle: "Constraints"
+weight: 10
+type: "docs"
+description: "Configure constraints to specify certain types of motion."
+---
+
+You can control the motion of your robot using the Motion Service's built-in constraint options.
+
+Available constraints all control the topological movement of the moving component along its path.
+
+The following constraints are available:
+
+- [Linear Constraint](#linear-constraint)
+- [Orientation Constraint](#orientation-constraint)
+
+## Linear Constraint
+
+The linear constraint (`{"motion_profile": "linear"}`) forces the path taken by `component_name` to follow an exact linear path from the start to the goal.

```

Comment:
You need to explain in the first sentence that this is a parameter passed to the Move method, to give context

Reply:
done

Reply:
I guess you do that at the top of the page

Reply:
I added that because of this comment :D